board/drivers/rtc.h

```diff
@@ -78,11 +78,11 @@ void rtc_set_time(timestamp_t time){
 
 timestamp_t rtc_get_time(void){
     timestamp_t result;
-    // Init with zero values in case there is no RTC running
-    result.year = 0U;
-    result.month = 0U;
-    result.day = 0U;
-    result.weekday = 0U;
+    // Init with correct starting values in case there is no RTC running(zeros will break python datetime)
```

Comment:
does it? if so, shouldn't we handle this in the python lib instead?

Reply:
Does, tested. Python crashes on datetime. 
My opinion that it should be handled in both places. 
If panda responds to rtc request - it should be the right one. And lib also should be protected (also noted to add this check to lib)

Reply:
this seems more like a bug in python tbh, I'd rather have the panda respond with all 0s if it's not valid

Reply:
Python's datetime lib will complain if values are wrong. But ok, won't argue on that, but still would like to reply with correct format )

Reply:
well, don't we want it to complain? seems more like a feature than a bug :)

Reply:
agree, think this should be handled in the python lib

Reply:
comment is out of date now